addons/storyshots/src/test-bodies.js

```diff
@@ -39,3 +43,63 @@ export function renderOnly({ story, context }) {
   const storyElement = story.render(context);
   reactTestRenderer.create(storyElement);
 }
+
+export const imageSnapshotWithOptions = ({ port = 6006, host = 'localhost', scheme = 'http' }) => {
```

Comment:
As I see there is no cohesion between other "bodies" in this module and the `imageSnapshot` (and it's a big one). I think worth extracting it into a separate file. `index.js` can just import it from there.

Reply:
Ok, did it.